Pod/Classes/Lifecycle/Behaviors/Nav-Bar-Hairline-Fade/NavBarHairlineFadeBehavior.swift

```diff
@@ -101,6 +110,13 @@ fileprivate extension NavBarHairlineFadeUpdater {
     func scrollViewDidScroll(_ scrollView: UIScrollView) {
         let shift = scrollView.contentInset.top + scrollView.contentOffset.y
         hairlineAlpha = shift.scaled(from: contentOffsetFadeRange, to: 0...1, clamped: true)
+        
+        #if DEBUG
+        if logsContentOffset {
+            // This prints the content y-offset, of the scrollview, truncated to a max of 3 decimals
+            print("\(NavBarHairlineFadeUpdater.self) content y-offset: \(Double(floor(1000*shift)/1000))")
```

Comment:
#nth use `String.init(format:)` with %-based formatting instead of math. That should let you omit the comment, since it should be self-documenting.

If you were keeping the math, you'd want to put spaces around math operators. Not sure why SwiftLint didn't catch that.